<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="8" hostname="DESKTOP-4LU25GP" name="com.test.BaseTest" tests="9" failures="0" timestamp="12 Oct 2020 06:44:56 GMT" time="0.011" errors="1">
  <testcase name="createDriver" time="0.011" classname="com.test.BaseTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.test.BaseTest.createDriver(BaseTest.java:107)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- createDriver -->
  <testcase name="verifyHomepagePageTest" time="0.000" classname="com.test.BaseTest">
    <skipped/>
  </testcase> <!-- verifyHomepagePageTest -->
  <testcase name="doLoginTest" time="0.000" classname="com.test.BaseTest">
    <skipped/>
  </testcase> <!-- doLoginTest -->
  <testcase name="searchtest" time="0.000" classname="com.test.BaseTest">
    <skipped/>
  </testcase> <!-- searchtest -->
  <testcase name="FilterSearch" time="0.000" classname="com.test.BaseTest">
    <skipped/>
  </testcase> <!-- FilterSearch -->
  <testcase name="getNames" time="0.000" classname="com.test.BaseTest">
    <skipped/>
  </testcase> <!-- getNames -->
  <testcase name="writeExcel" time="0.000" classname="com.test.BaseTest">
    <skipped/>
  </testcase> <!-- writeExcel -->
  <testcase name="SignOut" time="0.000" classname="com.test.BaseTest">
    <skipped/>
  </testcase> <!-- SignOut -->
  <testcase name="invsearchtest" time="0.000" classname="com.test.BaseTest">
    <skipped/>
  </testcase> <!-- invsearchtest -->
</testsuite> <!-- com.test.BaseTest -->
